SimDataFormats/Track/src/classes_def.xml

```diff
@@ -13,4 +13,11 @@
   <class name="edm::RefProd<std::vector<SimTrack> >"/>
   <class name="edm::Ref<std::vector<SimTrack>,SimTrack,edm::refhelper::FindUsingAdvance<std::vector<SimTrack>,SimTrack> >"/>
   <class name="edm::RefVector<std::vector<SimTrack>,SimTrack,edm::refhelper::FindUsingAdvance<std::vector<SimTrack>,SimTrack> >"/>
+  <class name="UniqueSimTrackId"/>
+  <class name="std::pair<unsigned int, EncodedEventId>"/>
```

Comment:
Given the type alias
https://github.com/cms-sw/cmssw/blob/93e2e3051588c8f931cb559201d76f9edafdb9ec/SimDataFormats/Track/interface/UniqueSimTrackId.h#L9
these two lines are the duplicates. Furthermore, the dictionary for `std::pair<unsigned int, EncodedEventId>` should be defined in the packaged defining `EncodedEventID`, i.e. `SimDataFormats/EncodedEventId`.